docs/conf/example-proposal.rst

```diff
@@ -0,0 +1,50 @@
+Create a catchy title that promises pratical lessons
+====================================================
+
+Abstract
+--------
+
+Incldue a brief story, typically two or three paragraphs, that describes your personal work experience with the topic. Make sure the abstract appeals to our audience of documentarians. We're suspicious of proposals that look like they belong at other conferences.
+
+Include a list of things that our audience can learn from your talk, such as:
+
+- Lessons that the audience can apply in their own work
+- Things that audiences should research further
+- Spoilers that provide hints about the talk
+
+Avoid walls of text.
+
+Who and Why
+-----------
+
+Use this opportunity to empathize with your audience. Help them think: "Oh yes, this talk could help me when I do X in my work!"
+
+Our audience creates documentation primarily for software. Given the variety of tools used for software documentation, we rarely accept talks that include tooling.
```

Comment:
"that include information about using specific tools."

Reply:
Or "information specific to certain tools" ?

Reply:
Or "talks that focus on a specific software tool" ?